web/src/components/Header/DropDown.jsx

```diff
@@ -0,0 +1,15 @@
+import "../Header.scss";
+import React from "react";
+export default function DropDown({ age, changeAge }) {
+  return (
+    <select
+      onChange={(e) => changeAge(e.target.value)}
+      className={age === "Adult" ? "select-adult" : "select-pg"}
```

Comment:
this will affect the font that mariam changed so this needs to be changed to this i believe
```suggestion
      className="select-pg"
```